lib/convert.js

```diff
@@ -20,6 +20,8 @@ function convert(objPath, gltfPath, options) {
     var ao = defaultValue(options.ao, false);
     var generateNormals = defaultValue(options.generateNormals, false);
     var optimizeForCesium = defaultValue(options.optimizeForCesium, false);
+    var textureCompressionOptions = defaultValue(options.textureCompressionOptions, undefined);
+    var preserve = defaultValue(options.preserve, true);
```

Comment:
Can you make it a CLI flag as well? For the purposes of obj2gltf preserve can be renamed to optimize.

It's fine if `textureCompressionOptions` is not in the CLI for now.